### PR TITLE
Add AWS download instructions for Kaptain

### PR DESCRIPTION
**Pull request by cneth**
NOTE: You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4093.s3-website-us-west-2.amazonaws.com/

Add the AWS download instructions for Kaptain.   This needs to go against Main because we're going to publish kaptain on AWS in the next few days.   I believe this change will then need to be added to the kudo-kubeflow repo so that we don't lose it going forward.   I don't believe it needs to go into the develop branch because AIUI Kaptain docs are not published from there.

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-82306

## Checklist

- [x] Test all commands and procedures, if applicable.
- [ x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by gvasquezvargas — COMMENTED** [review]
Kaptain downloads are only available to licensed customers. To get an evaluation license for Kaptain, please contact <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.

File: pages/dkp/kaptain/1.3.0/download/index.md [hunk 1]
```diff
@@ -9,8 +9,27 @@ enterprise: false
...
+docker run -it --rm -v $(pwd):/kaptain <container_image_name>
+```
+
+You will then see the following output:
+
+```sh
+Kaptain install package placed in the local directory. See install instructions at: https://docs.d2iq.com/dkp/kaptain/
+```
+
+You will now see the `kubeflow-1.4.0_1.3.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kommander, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```
Comment (chessahoekstra) [inline pages/dkp/kaptain/1.3.0/download/index.md:32]:
```suggestion
You will now see the `kubeflow-1.4.0_1.3.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] using these binaries. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
```

**Comment by chessahoekstra** [comment]
@cneth this looks like it is based against `develop` and not `main`

**Comment by cneth** [comment]
> @cneth this looks like it is based against `develop` and not `main`

Yup, you are correct!!! New PR incoming!

**Pull request closed without merging**